File: Back/src/Tcc.Domain/VistoriaImagem.cs
using System;
using System.ComponentModel.DataAnnotations;
using Tcc.Domain.Base;

namespace Tcc.Domain
{
    public class VistoriaImagem : BaseModel
    {
        public Guid VeiculoId { get; set; }
        public Veiculo Veiculo { get; set; }
        public string ImagemUrl { get; set; }
        public string ImagemBase64 { get; set; }
    }
}

File: Front/TccApp-Web/src/app/models/VistoriaImagem.ts
import { OrigemCadastroType, StatusCadastroType } from "@app/util/enums";
import { Associado } from "./Associado";
import { Veiculo } from "./Veiculo";

export interface VistoriaImagem {
  id: string;
  veiculoId: string;
  veiculo: Veiculo;
  imagemUrl: string;
  // ImagemBase64: string;
}